app/src/main/res/values/styles.xml

```diff
@@ -105,4 +105,27 @@
         <item name="colorAccent">@color/white</item>
     </style>
 
+    <!-- Session Feedback -->
+    <style name="FeedbackTitle">
```

Comment:
One suggestion.
I guess it is better to create `styles_feedback.xml` and put this styles.
Because styles is tend to fat in my experience.